tinygrad/codegen/linearizer.py

```diff
@@ -152,7 +152,8 @@ def linearize(self):
 
     # limit dims if we need to
     # TODO: broken, and doesn't really belong here
-    #if self.opts.global_max and self.opts.local_max: self.limit_dims_to_max(self.opts.global_max, self.opts.local_max)
+    # needs to be enabled for WEBGPU stable diffusion
+    if self.opts.device == "WEBGPU" and self.opts.global_max and self.opts.local_max: self.limit_dims_to_max(self.opts.global_max, self.opts.local_max)
```

Comment:
Can we reenable this everywhere? (separate PR)

Reply:
I put up a PR for this, tests pass.
It was commented out in the Token removal [PR](https://github.com/tinygrad/tinygrad/pull/1723/files)